### DISCUSSION TITLE
[Show and Tell] BlockJoy: One-Click Deploy Dedicated Archive Nodes to Baremetal Servers

### DISCUSSION THREAD

**Discussion by eullrich** [ethereum-optimism/developers#253] [Show and tell]
### Description

Hello OP community!  Eric here at [http://Blockjoy.com](https://www.blockjoy.com/) - happy to share what we have been working on to make it painless to launch dedicated nodes on baremetal machines.

- We built a hypervisor that enables one-click node deployments to baremetal machines. 
- As of writing we have support for OP Archive with Trace flags in productio.  We are exploring OP-Reth and OP-Erigon support.
- We do not use K8 or Docker - everything is built from the ground up in Reth for optimal performance. 
- Snapshotting service running daily to deploy new nodes near the tip.
- Focus on dedicated, unmetered nodes on baremetal machines.
- Run on any server - our managed servers or your own.
- Pre-configured with monitoring and alerting.
- System can handle updates and failover recovery. 
- We can easily support any custom image or forks. 
- Because everything runs on baremetal it is a fraction of the cost of cloud providers or traditional RPC providers.

Feel free to reach out to me: eric@blockjoy.com tg: erxc_u if you would like to test anything out.

<img width="1214" alt="image" src="https://github.com/ethereum-optimism/developers/assets/1896893/08f1657e-015a-4202-acae-af85fbfac761">
<img width="1246" alt="image" src="https://github.com/ethereum-optimism/developers/assets/1896893/23ffd41c-b7d4-4f32-9b25-446de0c67df9">
<img width="1156" alt="image" src="https://github.com/ethereum-optimism/developers/assets/1896893/340d30ae-dd00-4a98-b0e4-8944ee25c223">

**Comment by opfocus** [comment]
it's  very exciting！

**Comment by sbvegan** [comment]
Thank you for sharing @eullrich! The website and web application UI is very clean :)
